### DISCUSSION TITLE
Meteomodem Pilotsonde support

### DISCUSSION THREAD

**Discussion by miniarci** [dl9rdz/rdz_ttgo_sonde#302] [Q&A]
Hello all
I'm very happy to use RDZ. One lack of is Pilotsonde  http://www.meteomodem.com/pilotsonde.html . 
Maybe is possible or planned add this type of sonde??
If that help i can share with creator that type sonde to expand this project :)

**Comment by dl9rdz** [comment]
Should be possible and not too much work.
Modulation should be similar to other Meteomodem sonde, just different transmission speed.

Code space on the TTGO is probably the biggest issue.  Maybe I can save a bit with some reorganization, or I finally have to do the reorganization of the flash partition table that would create significantly more space...

**Reply by oe3jtb** [reply]
Pilot sondes have no serial number

**Comment by miniarci** [comment]
Great, sounds optimistic :) 
If you need that type sonde please send me private message details to shipment.
